blockchain/chain.go

```diff
@@ -1820,10 +1848,27 @@ func New(config *Config) (*BlockChain, error) {
 		return nil, err
 	}
 
-	bestNode := b.bestChain.Tip()
+	bestNode = b.bestChain.Tip()
```

Comment:
It doesn't really matter, but is this redundant? It made me stop and look at everything else but I don't see anything that would change Tip()?